_includes/field-description-vatamount.md

```diff
@@ -3,14 +3,15 @@
 {% assign currency = "SEK" %}
 {% endif %}
 {%- capture vatamount_text -%}
-The amount of VAT to charge the payer, entered in the lowest monetary unit of
-the selected currency. E.g.:&nbsp;
+The payment's VAT (Value Added Tax) `amount`, entered in the lowest monetary 
+unit of the selected currency. E.g.:&nbsp;
 **`10000`** = `100.00` {{ currency }},&nbsp;
 **`5000`** = `50.00` {{ currency }}.&nbsp;
 
-If the `amount` given includes VAT, `vatAmount` may be displayed for the user in
-the payment page (redirect only). Set to `0` (zero) if there is no VAT amount&nbsp;
-charged.
+The `vatAmount` entered will not affect the `amount` shown on the payment page, 
+which only shows the total `amount`. This field is used to specify how much of
+the total `amount` the VAT will be. Set to `0` (zero) if there is no VAT
+`amount` charged.
 {%- endcapture -%}
 {{- vatamount_text | strip_newlines | replace: "&nbsp;", " " -}}
```

Comment:
I only now saw this replacement of `&nbsp;` with ` ` and vaguely remember why I did it. Can you confirm that everything looks fine in the portal and that every word is separated by a space as it should?

Reply:
I'll ask @marinafrydenb if she can do it, since she has a working docker set up.

Reply:
![image](https://user-images.githubusercontent.com/43410587/95743882-c08a9c00-0c92-11eb-989f-ce02c70274d3.png)

Slik ser det ut nå!

Reply:
I checked and it looks good :)

Reply:
I think there's missing a space between the very last `VAT` and `amount`?